site/src/pages/DeploymentSettingsPage/IdpOrgSyncPage/PillList.tsx

```diff
@@ -0,0 +1,98 @@
+import { useTheme } from "@emotion/react";
+import { Pill } from "components/Pill/Pill";
+import {
+	Popover,
+	PopoverContent,
+	PopoverTrigger,
+} from "components/Popover/Popover";
+import type { FC } from "react";
+import { cn } from "utils/cn";
+
+interface PillListProps {
+	organizations: readonly string[];
+}
+
+// used to check if the organization is a UUID
+const UUID =
+	/^[0-9a-f]{8}-[0-9a-f]{4}-[0-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i;
```

Comment:
can I see your regex license please? 👮‍♀️

on a real note, if we're going to have something like this, it should definitely not be tucked away inside a component file. this needs to live somewhere more visible.